src/web/twig/InertiaExtension.php

```
@@ -24,23 +26,39 @@ public function getFilters()
         ];
     }
 
+    /**
+     * Outputs a JSON-like marker for controller parsing and stores the prop for controller collection.
+     */
+    public function prop(\Twig\Environment $env, $name, $value = null)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$env'. (Unused Code Rules)

(UnusedFormalParameter)